.github/workflows/check-dist-sha256.yml

```diff
@@ -0,0 +1,51 @@
+name: Check dist sha256
+# Validates whether the dist folder is as expected.
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+jobs:
+  check-dist:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+
+      - name: Set Node.js 16
+        uses: actions/setup-node@2fddd8803e2f5c9604345a0b591c3020ee971a93
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: |
+          cd .github/actions/compute-sha256
+          npm i
```

Comment:
please update to `npm ci`

Reply:
This has to be `npm i` to install. Why should it be `npm ci`? Can you please clarify?

Reply:
`npm i` does not honor the lockfile. If some dependencies are missing, they'll be fetched. `npm ci` guarantees that all dependencies have an `integrity` field. This is a check we have in the scorecard `Pinned-Dependency` check.

Reply:
OK, got it! I wasn't aware of this. I have updated it. Thanks